tests/unit/javascript/conftest.py

```diff
@@ -68,6 +78,38 @@ def javaScriptConsoleMessage(self, msg, line, source):
             """Fail tests on js console messages as they're used for errors."""
             pytest.fail("js console ({}:{}): {}".format(source, line, msg))
 
+if QWebEnginePage is None:
+    TestWebEnginePage = None
+else:
+    class TestWebEnginePage(QWebEnginePage):
+
+        """QWebPage subclass which overrides some test methods.
```

Comment:
QWebPage -> QWebEnginePage (or maybe just remove the subclass part, that's redundant - my bad!)